myscripts.js

```diff
@@ -0,0 +1,168 @@
+const menuBtn = document.querySelector('.openNav');
+const closeBtn = document.querySelector('.menu-list');
+const portfolioLink = document.querySelector('.portfolio-link');
+const aboutLink = document.querySelector('.about-link');
+const contactLink = document.querySelector('.contact-link');
+
+menuBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.getElementById('myLinks').style.width = '100%';
+  document.getElementById('myLinks').style.display = 'block';
+});
+
+closeBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  document.getElementById('myLinks').style.width = '0%';
+});
+
+portfolioLink.addEventListener('click', () => {
+  document.getElementById('myLinks').style.width = '0%';
+});
+aboutLink.addEventListener('click', () => {
+  document.getElementById('myLinks').style.width = '0%';
+});
+contactLink.addEventListener('click', () => {
+  document.getElementById('myLinks').style.width = '0%';
+});
+
+const cardContainer = document.querySelector('.project');
+const data = [
+  {
+    id: '1',
+    title: 'Profesional Art Printing Data',
+    text: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
```

Comment:
- [ ] Please add more lines to your text. Here is the link to [Lorem Ipsum](https://lipsum.co/) if you want to use that to get more lines of text
- [ ] Please make the page at the back of your popup less visible.
- Original Design:
![a](https://user-images.githubusercontent.com/63972670/174450309-69311995-9696-470b-a624-06953550733e.png)
 - Your Design
 
![b](https://user-images.githubusercontent.com/63972670/174450338-3c8467ad-55b5-410e-b5af-017f0c177705.png)